openedx/core/djangoapps/profile_images/tests/test_views.py

```diff
@@ -330,34 +335,12 @@ def check_remove_event_emitted(self):
             setting='profile_image_uploaded_at', old=TEST_UPLOAD_DT, new=None
         )
 
-    def test_unsupported_methods(self, mock_log):
-        """
-        Test that GET, PUT, PATCH, and DELETE are not supported.
-        """
-        self.assertEqual(405, self.client.get(self.url).status_code)
-        self.assertEqual(405, self.client.put(self.url).status_code)
-        self.assertEqual(405, self.client.patch(self.url).status_code)
-        self.assertEqual(405, self.client.delete(self.url).status_code)
-        self.assertFalse(mock_log.info.called)
-        self.assert_no_events_were_emitted()
-
-    def test_anonymous_access(self, mock_log):
```

Comment:
Nit:
I consider testing the absence of the unsupported_methods as a test for the View.  So, yes, I can see the rationale for testing that only once.
However, I consider verification of a REST method's access as a method-specific test and should be tested for all methods (POST and DELETE).